amd.test.ts

```diff
@@ -144,5 +152,38 @@ describe('Loader', function () {
     cleanUp();
   });
 
+  it('resolve relative deps', async () => {
+    const {System, fetcher} = setup();
+    fetcher.set('https://angul.ar/rxjs', readFileSync('./bundles/rxjs.js', 'utf-8'));
+    System.addImportMap({
+        imports: {
+            'rxjs': 'https://angul.ar/rxjs',
+        }
+    });
+
+    registerIndex(System);
```

Comment:
nit/optional, if those files are never gonna be used separately, consider combining the registration in one function